extensions/wikia/ArticleVideo/styles/ooyala/components/_ad-screen.scss

```diff
@@ -0,0 +1,24 @@
+.oo-ad-screen {
```

Comment:
How about commenting out unused stylesheets in `html5-skin.scss` with a comment saying that when we need them, we should uncomment? There is _a lot_ of these unused styles.